vmdb/db/migrate/20150414085027_create_container_routes.rb

```diff
@@ -0,0 +1,21 @@
+class CreateContainerRoutes < ActiveRecord::Migration
+  def up
+    create_table :container_routes do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :creation_timestamp
+      t.string     :resource_version
+      t.string     :namespace
+      t.string     :host
```

Comment:
> host is probably a bad column name. I'm concerned it will be confused with a reference to the host table at the model level

I'll change it to host name. (host is the original name of that field in openshift...)